CONTRIBUTING.md

```diff
@@ -37,10 +37,15 @@ Increment the version in the `src/main/resources/application.properties` and `po
 ## Deploying
 
 [For internal notify use only]
-Make sure your `~/.m2/settings.xml` file is up to date with the file found at `credentials/bintray/settings.xml` and you have updated the versions in the required files.
+You'll need to make sure you have the `notify-gpg-key` private key available locally.
+
+```shell
+gpg --import <(notify-pass show credentials/concourse/gpg-key)
```

Comment:
I'm very cautious if this is a manual deployed thing. 

I don't want a single GPG key to need to be imported into a developers key chain that is the key used by our CI system. The impact of impact of a compromise of that key would be much greater than that of a single key used for signing our java package.

If we stick with a manual process here then I'd be keen for us to have a separate GPG key used for it

Reply:
great point. will have a think. i'd love for this not be a manual deploy thing, i'm not against creating a separate GPG key either way but need to figure out if the fact we've already published a package with the shared GPG key affects anything

Reply:
Decided to go ahead with this for now, but I'll have a crack at getting concourse running this later